apps/playground/src/app/quiz/containers/question/question.component.html

```diff
@@ -0,0 +1,60 @@
+<mat-card>
+  <mat-card-header>
+    <div mat-card-avatar class="header-image"></div>
+      <mat-card-title>Dependency Injection Quiz</mat-card-title>
+      <mat-card-subtitle>
+        Assess your knowledge of Dependency Injection (<abbr>DI</abbr>)
+      </mat-card-subtitle>
+  </mat-card-header>
+
+  <mat-card-content>
+    <section class="scoreboard">

```

Comment:
Sounds like scoreboard should be a separate component?

Reply:
in progress

Reply:
trying to get everything mostly working before I start moving things around

Reply:
I made scoreboard its own component with score and timer children components, am working on the dependency injection with QuizService and TimerService.